### ISSUE TITLE
@UsingDataSet should throw an exception if a listed source file isn't found

### ISSUE THREAD

**Issue by chrylis** [lordofthejars/nosql-unit#143]
We just spent a couple of hours tracking down what turned out to be a typo in our `@UsingDataSet` directive. I suggest the framework should throw an exception if a resource specified there doesn't exist.

**Comment by lordofthejars** [comment]
Yes I agree with you about this. The problem is that now I am a bit busy preparing J1, Devoxx, and regular job, so I don't have time to improve it. But if you know where the code could be improved to show this, please don't hesitate to send a PR :)

Alex.

**Comment by chrylis** [comment]
I'll take a look. Is this in a common place, or is it handled independently by different providers?

**Comment by emas80** [comment]
Hi, I spent some time tracking the same issue .. I would really suggest to throw an exception.
I am using

**Comment by lordofthejars** [comment]
I get this on my prior list.
